lib/internal/test_runner/utils.js

```diff
@@ -409,6 +406,35 @@ const kColumns = ['line %', 'branch %', 'funcs %'];
 const kColumnsKeys = ['coveredLinePercent', 'coveredBranchPercent', 'coveredFunctionPercent'];
 const kSeparator = ' | ';
 
+function buildFileTree(summary) {
+  const tree = { __proto__: null };
+  let treeDepth = 0;
+  let longestFile = 0;
+
+  ArrayPrototypeForEach(summary.files, (file) => {
+    let longestPart = 0;
+    const parts = relative(summary.workingDirectory, file.path).split('/');
```

Comment:
```suggestion
    const parts = ArrayPrototypeSplit(relative(summary.workingDirectory, file.path), '/');
```

Primordials?

Reply:
done

Reply:
@RedYetiDev, thank you very much for reviewing this, it's greatly appreciated 😁

Reply:
P.S. @RedYetiDev: I left this PR in draft intentionally as it definitely needs a good dose of refactoring and a performance check 😁